grype/presenter/cyclonedx/test-fixtures/snapshot/TestCycloneDxPresenterDir.golden

```diff
@@ -1,83 +1,85 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<bom xmlns="http://cyclonedx.org/schema/bom/1.2" xmlns:v="http://cyclonedx.org/schema/ext/vulnerability/1.0" version="1" serialNumber="urn:uuid:f44893ea-988d-4ff8-bf21-e825ca710e77">
+<bom xmlns="http://cyclonedx.org/schema/bom/1.4" serialNumber="urn:uuid:01e40bfe-6f3d-4a97-8ac0-e47c8023d686" version="1">
   <metadata>
-    <timestamp>2022-11-23T16:21:11-05:00</timestamp>
+    <timestamp>2022-12-21T15:20:36-05:00</timestamp>
     <tools>
       <tool>
         <vendor>anchore</vendor>
         <name>grype</name>
         <version>[not provided]</version>
       </tool>
     </tools>
-    <component type="file">
-      <name>/some/path</name>
-      <version></version>
-    </component>
   </metadata>
   <components>
-    <component type="library">
+    <component bom-ref="76bd1479d016ce8f" type="library">
       <name>package-1</name>
       <version>1.1.1</version>
-      <v:vulnerabilities>
```

Comment:
Vulnerabilities are no longer embedded in components for 1.4 -- see latest schema definitions in PR description